scrapy/extensions/feedexport.py

```diff
@@ -356,32 +355,28 @@ def _close_slot(self, slot, spider):
             # properly closed.
             return defer.maybeDeferred(slot.storage.store, slot.file)
         slot.finish_exporting()
-        logfmt = "%s %%(format)s feed (%%(itemcount)d items) in: %%(uri)s"
-        log_args = {'format': slot.format,
-                    'itemcount': slot.itemcount,
-                    'uri': slot.uri}
+        logfmt = f"{slot.format} feed ({str(slot.itemcount)} items) in: {slot.uri}"
```

Comment:
As this is no longer a format string, it can be renamed to something like "logmsg" I think.

Reply:
@wRAR , fixed https://github.com/scrapy/scrapy/pull/5246/commits/91b776e47287ddcbce2f675041ef5b6366d748a8